docs/add-root-ca.rst

```diff
@@ -0,0 +1,45 @@
+Add a root CA to the web browser
+================================
+
+.. todo::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using an operationnal cluster.
+  Web browsers version used for tests:
+  Googe Chrome: Version 75.0.3770.80 | Mozilla Firefox: Version 67.0
```

Comment:
Why not a list formatting (it would be more readable).

Reply:
I tried to do a list, please tell me if it's better like this.